honeycombio/honeycomb-opentelemetry-node
# chore: add metrics tests to smoke tests

## Which problem is this PR solving?

- Closes #130 

## Short description of the changes

- add metrics to JS express app (already added to TS app)
- update counter to use "sheep" because we use it in other distros and smoke test for it ¯\_(ツ)_/¯
- remove metric interval from smoke tests (this caused issues with dotnet smoke testing, may be unnecessary)
- remove unused metric exporter package from our experiment when adding metrics exporter

## How to verify that this has the expected result

smoke tests pass!

If you run it locally you should see the output in `smoke-tests/collector/data-results/data-app-sdk-http-ts.json` and `smoke-tests/collector/data-results/data-app-sdk-http` that includes `resourceMetrics`.

In CircleCI, the data is also saved in [artifacts](https://app.circleci.com/pipelines/github/honeycombio/honeycomb-opentelemetry-node/410/workflows/35504d47-5f63-4ffb-923d-3e8baaa41483/jobs/1629/artifacts).
